phpunit.xml.dist

```diff
@@ -124,5 +124,10 @@
             <directory>vendor/silverstripe/login-forms/tests</directory>
         </testsuite>
 
+        <testsuite name="other">
+            <directory>vendor/symbiote/silverstripe-gridfieldextensions/tests</directory>
+            <directory>vendor/silverstripe/silverstripe-staticpublishqueue/tests</directory>
+        </testsuite>
```

Comment:
silverstripe/lumberjack is already in the recipe-blog testsuite above
colymba/gridfield-bulk-editing-tools has no unit tests